pkg/app/piped/cloudprovider/cloudrun/cloudrun.go

```diff
@@ -44,9 +44,28 @@ type (
 type Client interface {
 	Create(ctx context.Context, sm ServiceManifest) (*Service, error)
 	Update(ctx context.Context, sm ServiceManifest) (*Service, error)
+	List(ctx context.Context, params *ListServicesParams) ([]*Service, string, error)
 	GetRevision(ctx context.Context, name string) (*Revision, error)
 }
 
+type ListServicesParams struct {
+	Limit         int64
+	LabelSelector string
+	PageSize      string
```

Comment:
Actually, this is not a page size number but a token to fetch the next data, right?
So how about naming it as "cursor"?

Reply:
Absolutely! I'll fix it! 
TYVM :)